perception_dataset/deepen/segmentation/painting2d.py

```diff
@@ -141,8 +141,9 @@ def _parse_annotation(
         for image, categories in items.items():
             id2category = {i + 1: name for i, name in enumerate(categories)}
             camera_name = None
+            # NOTE: camera2index starts from 0, but deepen sensor id starts from 1.
             for name, sensor_idx in camera2index.items():
-                if f"sensor{sensor_idx}" == sensor_id:
+                if f"sensor{sensor_idx + 1}" == sensor_id:
```

Comment:
@ktro2828 

[must]

I had a trouble with this line.

```bash
❯ python3 -m perception_dataset.convert --config config/convert_deepen_to_t4_segmentation_painting_sample.yaml --overwrite

[2024-11-16 15:55:59,537] [INFO] [process] 2063996 MainProcess [thread] 140220525953024 MainThread [file] /home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/convert.py [func] main [line] 187 [[BEGIN] Converting Deepen data (./data/sample_segmentation_paints/non_annotated_dataset) to T4 data (./data/t4_format)]
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/convert.py", line 395, in <module>
    main()
  File "/home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/convert.py", line 188, in main
    converter.convert()
  File "/home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/deepen/deepen_to_t4_converter.py", line 75, in convert
    else DeepenSegmentationPainting2D.from_file(
  File "/home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/deepen/segmentation/painting2d.py", line 53, in from_file
    annotations = _parse_annotation(ann_file, camera2index)
  File "/home/user/src/github.com/tier4/tier4_perception_dataset/perception_dataset/deepen/segmentation/painting2d.py", line 149, in _parse_annotation
    raise ValueError(f"There is no corresponding camera for {image}")
ValueError: There is no corresponding camera for data_CAM_TRAFFIC_LIGHT_NEAR_00000.jpg
```

How about adding subtract like [this](https://github.com/tier4/tier4_perception_dataset/pull/164#discussion_r1840104294)?